controlador.py

```diff
@@ -0,0 +1,62 @@
+from snake import Snake
+from apple import Apple
+import pygame
+from pygame.locals import *
+
+"""Functions"""
+
+
+def colision(c1, c2):
+    return (c1[0] == c2[0]) and (c1[1] == c2[1])
+
+
+"""Colors"""
+
+BLACK = 0, 0, 0
+WHITE = 255, 255, 255
+RED = 255, 0, 0
+
+
+pygame.init()
+
+"""Screen"""
+SCREEN = pygame.display.set_mode((600, 600))
+pygame.display.set_caption('Snake Game')
+
+"""Pygame Objects"""
+
+clock = pygame.time.Clock()  # this will be our clock object
+snake_obj = Snake()  # this will be our snake object
+apple_obj = Apple()
+apple_obj.onGrid_Random_Spawn()  # position get
+run = True
+
+# Game loop
+while run:
+    clock.tick(10)  # our clock object will set the game fps
+    snake_obj.move()
+    for event in pygame.event.get():#  event catcher for inputs
+        if event.type == QUIT:
+            pygame.quit()
+        if event.type == KEYDOWN:
+            if event.key == K_UP and snake_obj.snake_direction != snake_obj.DOWN:
+                snake_obj.snake_direction = snake_obj.UP
+            if event.key == K_DOWN and snake_obj.snake_direction != snake_obj.UP:
+                snake_obj.snake_direction = snake_obj.DOWN
+            if event.key == K_LEFT and snake_obj.snake_direction != snake_obj.RIGHT:
+                snake_obj.snake_direction = snake_obj.LEFT
+            if event.key == K_RIGHT and snake_obj.snake_direction != snake_obj.LEFT:
+                snake_obj.snake_direction = snake_obj.RIGHT
+
+    if colision(snake_obj.snake_body[0], apple_obj.position):
+        apple_obj.onGrid_Random_Spawn()
```

Comment:
O que acontece perto do final do jogo, se a cobra for muito grande e a maçã renascer no meio do corpo dela?

Reply:
mas pra frente resolvo isso